src/components/Modal/Modal.js

```diff
@@ -131,7 +137,7 @@ export default class Modal extends Component {
     return (
       <div
         {...other}
-        onKeyDown={this.handleKeyDown}
+        onKeyDown={event => this.handleKeyDown(event, enterSubmit)}
```

Comment:
Could we leave this as `this.handleKeyDown`, and then just read from props in that handler?

Reply:
Yup, this was my original idea but the ESLint kept yelling about the prop not being used. I believe it is because I don't actually use that prop inside of the render function, just the helper.